### PR TITLE
Write unit tests for message handler

### LINKED ISSUE #269: Write unit tests for message handler
# Summary

Unit tests do not currently exist for the message handler. 

# Background

**Is your improvement related to a problem? Please describe**:

After writing the message handler, it is unclear if we have handled all edge cases. We should have unit tests to thoroughly test the message handler functionality.

**Describe the solution you'd like**:

Full set of unit tests for the message handler to catch as many edge cases as we can.

**Describe alternatives you've considered**:

No alternatives.

# Details

The owner of this issue needs to think of what the message handler should support, and not support. Then, build unit tests around the desired and undesired functionality to make sure error handling exists in the appropriate locations.

# Outcome

Unit testing encompasses the message handler. Allows us to feel safe knowing the message handler has been thoroughly tested.

### PR DESCRIPTION
**Pull request by Zedjones**
Added unit tests for the message handler and moved a string concatenation to the inside of a check since it was unused if the check failed.

Resolves #269

### CODE REVIEW

File: internal/handlers/irc/handlers_test.go [hunk 1]
```diff
@@ -314,3 +314,120 @@ func TestKickHandlerNoReason(t *testing.T) {
...
+}
+
+func TestMessageHandlerFull(t *testing.T) {
+	ctrl := gomock.NewController(t)
+
+	defer ctrl.Finish()
+
+	ircSettings := internal.IRCSettings{
+		IRCBlacklist: []string{},
+		Prefix:       "<<",
```
Comment (Tjzabel) [inline internal/handlers/irc/handlers_test.go:399]: Prefix & Suffix are just `"<"`, not the double `<<|>>`.

Comment (Zedjones) [reply internal/handlers/irc/handlers_test.go:462]: Resolved during meeting

**Comment by jwflory** [comment]
In the developer meeting today, we decided to go ahead with this test as-is as another test case. Merge conflict fixed and tests passing. Merging. :clapper:
